lib/index.js

```diff
@@ -58,13 +58,23 @@ module.exports = {
           }
         }
 
+        const executable = (
+          this.executablePath ||
+          await findCachedAsync(fileDirectory, 'node_modules/.bin/flow') ||
```

Comment:
IMO the order of working should be

0. Use the project local version if there is any or
0. Use the version in the config as a fallback or
0. fallback to the one in PATH, if there is any